geopy/geocoders/here.py

```diff
@@ -0,0 +1,281 @@
+
+        Documentation at:
+            https://developer.here.com/documentation/geocoder/topics/resource-reverse-geocode.html
+
+        :param query: The coordinates for which you wish to obtain the
+            closest human-readable addresses.
+        :type query: :class:`geopy.point.Point`, list or tuple of ``(latitude,
+            longitude)``, or string as ``"%(latitude)s, %(longitude)s"``.
+
+        :param bool exactly_one: Return one result or a list of results, if
+            available.
+
+        :param int timeout: Time, in seconds, to wait for the geocoding service
+            to respond before raising a :class:`geopy.exc.GeocoderTimedOut`
+            exception. Set this only if you wish to override, on this call
+            only, the value set during the geocoder's initialization.
+
+        :param str language: Affects the language of the response,
+            must be a RFC 4647 language code, e.g. 'en-US'.
+
+        :param dict other: A dictionary with additional key/values as described
+            in the online documentation. These will be added as individual query
+            parameters to the URL.
+
+        :rtype: ``None``, :class:`geopy.location.Location` or a list of them, if
+            ``exactly_one=False``.
+        """
+        point = self._coerce_point_to_string(query)
+        params = {
+            'app_id': self.app_id,
+            'app_code': self.app_code,
+            'maxresults': 1,
+            'mode': 'retrieveAddresses',  # can be overwritten using `other`
+            'prox': point + ',1000',
+        }
+        if exactly_one:
+            params['maxresults'] = 1
+        if language:
+            params['language'] = language
+        if other:
+            params.update(other)
+
+        url = "%s?%s" % (
+            self.reverse_api, urlencode(params))
+        logger.debug("%s.reverse: %s", self.__class__.__name__, url)
+        return self._parse_json(
+            self._call_geocoder(url, timeout=timeout),
+            exactly_one
+        )
+
+    @staticmethod
+    def _parse_json(doc, exactly_one=True):
+        """
+        Parse a location name, latitude, and longitude from an JSON response.
+        """
+        status_code = doc.get("statusCode", 200)
+        if status_code != 200:
+            err = doc.get("errorDetails", "")
+            if status_code == 401:
+                raise GeocoderAuthenticationFailure(err)
+            elif status_code == 403:
+                raise GeocoderInsufficientPrivileges(err)
+            elif status_code == 429:
+                raise GeocoderQuotaExceeded(err)
+            elif status_code == 503:
+                raise GeocoderUnavailable(err)
+            else:
+                raise GeocoderServiceError(err)
+
+        try:
+            resources = doc['Response']['View'][0]['Result']
+        except IndexError:
+            resources = None
+        if resources is None or not len(resources):
```

Comment:
I think this is equal to `if not resources:` in this case.

Reply:
Sounds right. Will change, if so.